app/helpers/image_conversion_helper.rb

```diff
@@ -1,85 +1,37 @@
+require "wasmer"
 module ImageConversionHelper
   IMAGE_QUALITY = 50
 
   def resize_to_dimensions_and_convert_to_jpg(source_image_path:, attachment_type:, filename:)
-    # Set dimensions
-    mini_magick_image = MiniMagick::Image.open(source_image_path)
+    file = File.binread(source_image_path)
+    file_bytes = file.unpack("C*")
+
+    dimensions = nil
     if attachment_type == SiteBanner::LOGO
-      mini_magick_image.resize(SiteBanner::LOGO_DIMENSIONS.join("x"))
+      dimensions = SiteBanner::LOGO_DIMENSIONS
     elsif attachment_type == SiteBanner::BACKGROUND
-      mini_magick_image.resize(SiteBanner::BACKGROUND_DIMENSIONS.join("x"))
-    else
+      dimensions = SiteBanner::BACKGROUND_DIMENSIONS
+    end
+
+    if !dimensions
       LogException.perform(StandardError.new("Unknown attachment type:" + attachment_type), params: {})
       return nil
     end
 
-    # Use JPG as its the most efficient standard
-    mini_magick_image.format "jpg"
+    actual_jpeg_to_save = Wasm::Thumbnail::Rb.resize_and_pad(file_bytes: file_bytes,
+                                                             width: dimensions[0],
+                                                             height: dimensions[1],
+                                                             size: 250000)
```

Comment:
do we want this to be `250000 + 4`? since the output buffer would be `250000` bytes before we chop the first 4 off containing the length

Reply:
The gem takes care of this now.